packages/opentelemetry-tracing/src/export/SimpleSpanProcessor.ts

```diff
@@ -27,16 +27,29 @@ import { Span } from '../Span';
  */
 export class SimpleSpanProcessor implements SpanProcessor {
   constructor(private readonly _exporter: SpanExporter) {}
+  private _isShutdown = false;
+
+  forceFlush(): void {
+    // do nothing as all spans are being exported without waiting
+  }
 
   // does nothing.
   onStart(span: Span): void {}
 
   onEnd(span: Span): void {
+    if (this._isShutdown) {
```

Comment:
This looks redundant to me, we already have the same logic in [Zipkin](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-exporter-zipkin/src/zipkin.ts#L75) and [Collector](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-exporter-collector/src/CollectorExporter.ts#L79) exporter. Maybe bug in Stackdriver exporter where we don't do anything on shutdown and in Jaeger exporter, we juse close the socket connection.

Reply:
I actually agree. if the span processor shuts down its exporter, then it should be safe for it to continue sending spans even after shutdown and the exporter will just no-op

Reply:
but this is what spec says for span processors
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-tracing.md#shutdown
so they should have the same functionality

Reply:
one additional `if` check isn't hurting anything. I would say just leave it in